supporting_scripts/analysis-of-endpoint-connections/analysisOfEndpointConnections.config.yml

```diff
@@ -0,0 +1,282 @@
+  - 'api/programming-exercises/{exerciseId}/solution-entries/{solutionEntryId}'
+  - 'api/programming-exercises/{exerciseId}/code-hints/{codeHintId}/solution-entries'
+  - 'api/programming-exercises/{exerciseId}/test-cases/{testCaseId}/solution-entries'
+  - 'api/programming-exercises/{exerciseId}/code-hints'
+  - 'api/programming-exercises/{exerciseId}/tasks-with-unassigned-test-cases'
+  - 'api/programming-exercises/{exerciseId}/testwise-coverage-report'
+  - '.well-known/assetlinks.json'
+  - 'api/exercises/{exerciseId}/title'
+  - 'api/participations/{participationId}/withLatestResult'
+  - 'api/aeolus/templates/{language}/{projectType}'
+  - 'api/aeolus/templateScripts/{language}/{projectType}'
+  - 'api/lectures/{lectureId}/title'
+  - 'api/exams/active'
+  - 'api/exams'
+  - 'api/exams/{examId}/title'
+  - 'api/courses/{courseId}/exams/{examId}/exam-for-assessment-dashboard'
+  - 'api/courses/{courseId}/exams/{examId}/exam-for-test-run-assessment-dashboard'
+  - 'api/courses/{courseId}/exams/{examId}/download-archive'
+  - '.well-known/jwks.json'
+  - 'api/public/authenticate'
+  - 'api/public/pyris/pipelines/tutor-chat/runs/{runId}/status'
+  - 'api/public/pyris/pipelines/course-chat/runs/{runId}/status'
+  - 'api/public/pyris/webhooks/ingestion/runs/{runId}/status'
+  - 'api/public/programming-submissions/{participationId}'
+  - 'api/public/programming-exercises/test-cases-changed/{exerciseId}'
+  - 'api/public/programming-exercises/{exerciseId}/build-plan'
+  - 'api/public/time'
+  - 'api/public/programming-exercises/new-result'
+  - 'api/public/imprint'
+  - 'api/programming-exercises/{exerciseId}/submission-policy'
+  - 'api/data-exports/{dataExportId}'
+
+  # Issue #9377
+  - 'LTI13_LOGIN_REDIRECT_PROXY_PATH'
+  - 'LTI13_DEEPLINK_REDIRECT_PATH'
+
+excludedRestCallFiles:
+  # Files that only contain REST calls that are not yet parsed correctly
+  - 'src/main/webapp/app/assessment/athena.service.ts'
+  - 'src/main/webapp/app/shared/statistics-graph/statistics.service.ts'
+  - 'src/main/webapp/app/shared/server-date.service.ts'
+  - 'src/main/webapp/app/shared/image/cacheable-image.service.ts'
+  - 'src/main/webapp/app/shared/http/file.service.ts'
+  - 'src/main/webapp/app/lecture/lecture-paging.service.ts'
+  - 'src/main/webapp/app/exercises/shared/manage/exercise-paging.service.ts'
+  - 'src/main/webapp/app/exercises/shared/example-submission/example-submission-import/example-submission-import-paging.service.ts'
+  - 'src/main/webapp/app/exercises/programming/manage/services/build-plan.service.ts'
+  - 'src/main/webapp/app/exam/manage/exams/exam-import/exam-import-paging.service.ts'
+
+excludedRestCalls:
+  # The Endpoints matching the following REST calls are provided by SpringBootActuator and spring,
+  # see src/main/resources/config/application.yml management.endpoints.web.exposure.include
+  - 'management/configprops'
+  - 'management/env'
+  - '/management/inf'
+  - 'management/info'
```

Comment:
Is this duplication with the slash intended?

Reply:
Yes. they are 2 individual REST calls by the webapp.